data-serving/scripts/converters.py

```diff
@@ -343,14 +348,89 @@ def convert_outbreak_specifics(id: str, reported_market_exposure: str,
         normalized = convert_bool(lives_in_wuhan)
         if normalized is not None:
             outbreak_specifics['livesInWuhan'] = normalized
-    except (ValueError):
+    except ValueError:
         logging.warning(
             '[%s] [outbreakSpecifics[livesInWuhan]] invalid value %s', id,
             lives_in_wuhan)
 
     return outbreak_specifics if outbreak_specifics else None
 
 
+def convert_location_string(location_str: str) -> Dict[str, Any]:
+    if pd.isna(location_str) or location_str.lower() == 'no' or location_str.lower() == 'none':
```

Comment:
Are these two values the bulk of "str-as-no-value" that we're seeing? Mostly curious; I know this PR was advertised as low precision.

Reply:
yeah, I have a colab where I run value_counts() on the columns to see what the most frequent values are and try to account for those